Calibration/PPSAlCaRecoProducer/test/test_express_AlCaRecoProducer.sh

```diff
@@ -1,29 +1,32 @@
 #!/bin/bash
 function die { echo $1: status $2; exit $2; }
 
-customise_commands="process.GlobalTag.toGet = cms.VPSet()\
-\nprocess.GlobalTag.toGet.append(cms.PSet(record = cms.string(\"AlCaRecoTriggerBitsRcd\"),tag =  cms.string(\"AlCaRecoHLTpaths_PPS2022_express_v1\"), connect = cms.string(\"frontier://FrontierProd/CMS_CONDITIONS\")))\
-\nprocess.GlobalTag.toGet.append(cms.PSet(record = cms.string(\"PPSTimingCalibrationLUTRcd\"),tag =  cms.string(\"PPSDiamondTimingCalibrationLUT_test\"), connect = cms.string(\"frontier://FrontierProd/CMS_CONDITIONS\")))\
-\nprocess.ALCARECOPPSCalMaxTracksFilter.TriggerResultsTag = cms.InputTag(\"TriggerResults\",\"\",\"HLTX\")"
+# customisation command needed for all tests cases
+COMMON_CUSTOM="process.ALCARECOPPSCalMaxTracksFilter.TriggerResultsTag = cms.InputTag(\"TriggerResults\",\"\",\"HLTX\")"
 
-INPUTFILE="/store/group/alca_global/pps_alcareco_producer_tests/outputALCAPPS_single.root"
-COMMMAND=`xrdfs cms-xrd-global.cern.ch locate $INPUTFILE`
-STATUS=$?
-echo "xrdfs command status = "$STATUS
-if [ $STATUS -eq 0 ]; then
-    echo "Using file ${INPUTFILE}. Running in ${SCRAM_TEST_PATH}."
+# test on 2022 data
+INPUTFILE_355207="file:/eos/cms/store/user/cmsbuild/store/data/Run2022B/AlCaPPS/RAW/v1/000/355/207/00000/c23440f4-49c0-44aa-b8f6-f40598fb4705.root"
```

Comment:
```suggestion
INPUTFILE_355207="/store/user/cmsbuild/store/data/Run2022B/AlCaPPS/RAW/v1/000/355/207/00000/c23440f4-49c0-44aa-b8f6-f40598fb4705.root"
```

@grzanka @perrotta 
with this change the unit tests run fine [*] after setting the environment variables as in the central integration tests:

```
setenv CMS_PATH "/cvmfs/cms-ib.cern.ch"
setenv SITECONFIG_PATH "/cvmfs/cms-ib.cern.ch/SITECONF/local"
```

[*]
```console
$ scram b runtests
>> Local Products Rules ..... started
>> Local Products Rules ..... done
Creating test log file logs/el9_amd64_gcc12/testing.log
Pass  103s ... Calibration/PPSAlCaRecoProducer/testExpressPPSAlCaRecoProducer
Pass   17s ... Calibration/PPSAlCaRecoProducer/testExpressPPSAlCaRecoOutput
Pass   74s ... Calibration/PPSAlCaRecoProducer/testPromptPPSAlCaRecoProducer
Pass   16s ... Calibration/PPSAlCaRecoProducer/testPromptPPSAlCaRecoOutput
```

Reply:
Thanks for advice, that config with `SITECONFIG_PATH` and `CMS_PATH` is very useful. I think that I will add a readme.md explaning how to use it, reducing it to oneliner:

```
CMS_PATH="/cvmfs/cms-ib.cern.ch" SITECONFIG_PATH="/cvmfs/cms-ib.cern.ch/SITECONF/local" scram b runtests
```

Is this re-directed feature somewhere documented ?